src/domains/aca-app/api/aca/v1/_openapi.json.tpl

```diff
@@ -0,0 +1,218 @@
+            "description": "New debt position successfully created"
+          },
+          "400": {
+            "description": "Formally invalid input",
+            "content": {
+              "application/json": {
+                "schema": {
+                  "$ref": "#/components/schemas/ProblemJson"
+                },
+                "example": {
+                  "value": {
+                    "type": "https://example.com/problem/",
+                    "title": "string",
+                    "status": 400,
+                    "detail": "Formally invalid input"
+                  }
+                }
+              }
+            }
+          },
+          "404": {
+            "description": "PA or IBAN not found",
+            "content": {
+              "application/json": {
+                "schema": {
+                  "$ref": "#/components/schemas/ProblemJson"
+                },
+                "example": {
+                  "value": {
+                    "type": "https://example.com/problem/",
+                    "title": "string",
+                    "status": 404,
+                    "detail": "PA or IBAN not found"
+                  }
+                }
+              }
+            }
+          },
+          "409": {
+            "description": "Conflict during the debt position census",
+            "content": {
+              "application/json": {
+                "schema": {
+                  "$ref": "#/components/schemas/ProblemJson"
+                },
+                "example": {
+                  "value": {
+                    "type": "https://example.com/problem/",
+                    "title": "string",
+                    "status": 409,
+                    "detail": "Conflict during the debt position census"
+                  }
+                }
+              }
+            }
+          },
+          "500": {
+            "description": "Internal server error",
+            "content": {
+              "application/json": {
+                "schema": {
+                  "$ref": "#/components/schemas/ProblemJson"
+                },
+                "example": {
+                  "value": {
+                    "type": "https://example.com/problem/",
+                    "title": "string",
+                    "status": 500,
+                    "detail": "Internal server error"
+                  }
+                }
+              }
+            }
+          }
+        }
+      }
+    }
+  },
+  "components": {
+    "schemas": {
+      "NewDebtPositionRequest": {
+        "type": "object",
+        "required": [
+          "paFiscalCode",
+          "entityType",
+          "entityFiscalCode",
+          "entityFullName",
+          "iuv",
+          "amount",
+          "description",
+          "expirationDate"
+        ],
+        "description": "Request body for creating a new transaction",
+        "properties": {
+          "paFiscalCode": {
+            "type": "string"
```

Comment:
Add a brief description for each request fields

Reply:
Fixed with 17cec1bb7b114731d5cbfd78a517cd43a1abacbf 🙂

Reply:
Fixed with be0ebd08a30e1463f6f7e0df0803d9d46b88ae25 🙂